src/main/java/atas/model/session/VersionedAttributesList.java

```diff
@@ -15,6 +15,7 @@
     private final ObservableList<Attributes> currentAttributeList;
     private final List<List<Attributes>> attributeStateList;
     private int currentStatePointer;
+    private final List<String> tracker;
```

Comment:
Perhaps a comment could be added to provide a brief description as to what tracker is for? The name isn't as clear atm.